README.md

```diff
@@ -1 +1,12 @@
-2020-feb-project1-group3
\ No newline at end of file
+2020-feb-project1-group3
+
+# Diagram 
+
+[PR1-grp3](images/PR1-grp3.png)
```

Comment:
attach the image to the documentation so you don't need to click to see it

`![<some-alt-text>](images/PR1-grp3.png)`

Reply:
and please give me a few words for each one of the elements of the diagram